pkgs/applications/misc/collision/default.nix

```
@@ -0,0 +1,63 @@
+{ lib
+, fetchFromGitHub
+, callPackage
+, crystal
+, libadwaita
+, openssl
+, libxml2
+, pkg-config
+, wrapGAppsHook4
+}:
+let
+  gi_crystal = callPackage gi-crystal/default.nix { };
+in
+crystal.buildCrystalPackage rec {
+  pname = "Collision";
+  version = "3.5.0";
+
+  src = fetchFromGitHub {
+    owner = "GeopJr";
+    repo = pname;
```

```suggestion
    repo = "Collision";
```

See: https://github.com/nix-community/nixpkgs-lint/issues/21